components/vni.js

```diff
@@ -0,0 +1,23 @@
+// vni.js
+
+var _ = require('underscore');
+
+var outputStates = require('./outputStates');
+var inputStates = require('./inputStates');
+
+// Hash of vnis
+vnis = []; 
```

Comment:
The vnis array should normally be stored in nodeInstance.rpf.vnis , so I don't know what the intent is here.  Is this just temporary for stub purposes?